src/elements/kw-share-card/kw-share-card.html

```diff
@@ -150,7 +157,7 @@
                             <p>[[share.description]]</p>
                         </div>
                         <div class="actions">
-                            <button type="submit" class="orangebutton remixbutton" onclick="">remix<img src="../../assets/icons/shares/remix.svg" /></button>
+                            <button class="orangebutton remixbutton" onclick="">remix<img src="../../assets/icons/shares/remix.svg" /></button>
```

Comment:
You can remove the onclick too. And same as previous comment. link the src form the root